src/NServiceBus.Transport.RabbitMQ/RabbitMQTransport.cs

```diff
@@ -199,6 +205,15 @@ public TimeSpan NetworkRecoveryInterval
             }
         }
 
+        /// <summary>
+        /// Specifies whether the endpoint should ignore failures to declare the incoming queues because the queue already exists with a different configuration (e.g. arguments).
+        /// This is set to <code>false</code> by default which will make the endpoint fail to start when the queue configuration does not match an existing queue.
+        /// This option has no effect if installers have been disabled.
+        /// </summary>
+        public bool AllowInputQueueConfigurationMismatch { get; set; } = false;
```

Comment:
Since the only thing we allow to mismatch is what we call `QueueMode` should we rename it to ``AllowInputQueueModeMismatch` to make it more clear?

Reply:
technically this can mismatch on any type of argument mismatch between existing queue and queue created by the installers. It should be about the queue time 99% of the cases I guess.